src/components/SignUpForm.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from "react";
+import { Button, Form, Container, Message } from "semantic-ui-react";
+import { auth } from "../modules/auth";
+import { useHistory } from "react-router-dom";
+
+const SignUpForm = () => {
+  const [failureMessage, setFailureMessage] = useState("");
+  const history = useHistory();
+
+  const signUp = async (event, history) => {
+    event.preventDefault();
+    try {
+      const name = event.target.name.value;
+      const email = event.target.email.value;
+      const password = event.target.password.value;
+      const password_confirmation = event.target.passwordConfirmation.value;
+
+      const response = await auth.signUp({
+        name: name,
+        email: email,
```

Comment:
why to send it as an object?

Reply:
this one was just to know. No need to change it.

Reply:
This is how we have to send the parameters to the backend for sign up